src/com/google/enterprise/adaptor/sharepoint/SharePointAdaptor.java

```diff
@@ -2618,20 +2593,13 @@ private long addMetadata(Response response, String name, String value,
           .setPermits(permits);
     }
 
-    private void addPermitUserToAcl(int userId, Acl.Builder aclToUpdate)
+    private void addPermitUserToAcl(Integer userId, Acl.Builder aclToUpdate)
```

Comment:
You could look at this to optimize the boxing and unboxing that occurs. If you haven't done that, then it's probably not worth changing. I think it may be a tiny lose, and point out that this value is an int before it is passed in here, and is unboxed immediately on the first line of this method.